PWGDQ/Tasks/dqEfficiency.cxx

```diff
@@ -672,6 +679,13 @@ struct AnalysisSameEventPairing {
         VarManager::FillPairVertexing<TPairType, TEventFillMap, TTrackFillMap>(event, t1, t2, VarManager::fgValues);
       }
 
+      dileptonFilterMap = twoTrackFilter;
+      dileptonList(event, VarManager::fgValues[VarManager::kMass], VarManager::fgValues[VarManager::kPt], VarManager::fgValues[VarManager::kEta], VarManager::fgValues[VarManager::kPhi], t1.sign() + t2.sign(), dileptonFilterMap);
```

Comment:
I suggest to replace the Eta with the rapidity and then remove the rapidity member from the DileptonsExtra table.